src/apps/miscapps/generateBaseCellNeighbors.c

```diff
@@ -223,11 +237,14 @@ void generate() {
 
     printf("const int baseCellNeighbors[NUM_BASE_CELLS][7] = {\n");
     for (int i = 0; i < NUM_BASE_CELLS; i++) {
-        printf("{%d, %d, %d, %d, %d, %d, %d}, // base cell %d%s\n",
-               baseCellNeighbors[i][0], baseCellNeighbors[i][1],
-               baseCellNeighbors[i][2], baseCellNeighbors[i][3],
-               baseCellNeighbors[i][4], baseCellNeighbors[i][5],
-               baseCellNeighbors[i][6], i,
+        printf("{%s, %s, %s, %s, %s, %s, %s}, // base cell %d%s\n",
```

Comment:
I would prefer we didn't leak the strings allocated on line 85. Could this be simply changed as:

```
char[7]* neighborStrings;
getBaseCellOutput(baseCellNeighbors[i], neighborStrings);
// printf
destroyBaseCellOutput(neighborStrings);
```

Reply:
Sure, will make sure i486SX machines won't have any issues running this application. ;)